app/assets/javascripts/fae/form/_ajax.js

```diff
@@ -78,6 +79,9 @@ Fae.form.ajax = {
       Fae.form.text.initMarkdown();
       Fae.form.checkbox.setCheckboxAsActive();
 
+      // validate nested form fields on submit
+      Fae.form.validator.formValidate($('.nested-form'));
```

Comment:
This jQuery element is referenced earlier in the closure - it should be cached and hoisted if it's being called twice.